app/views/config_reports/_list.html.erb

```diff
@@ -5,6 +5,7 @@
       <th><%= sort :host, :as => _("Host") %></th>
       <% end %>
       <th><%= sort :reported, :as => _("Last report") %></th>
+      <th><%= _('Origin') %></th>
```

Comment:
could you please limit width so it does not take so much space? col-md-1 seems better locally